.github/workflows/docker_publish.yml

```diff
@@ -22,6 +22,22 @@ jobs:
         with:
           username: ${{ secrets.CADENCE_WEB_DOCKERHUB_USERNAME }}
           password: ${{ secrets.CADENCE_WEB_DOCKERHUB_TOKEN }}
-          repository: ubercadence/web
+          repository: justatuber/web
+          tag_with_ref: true
+          tags: latest
+  push_to_ghcr:
+    name: Push Docker image to GitHub Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Container Registry
+        if: "!github.event.release.prerelease"
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.CADENCE_WEB_GHCR_PAT }}
```

Comment:
Need to change back to `GITHUB_TOKEN` once github supports this.
See [here](https://github.community/t/github-packages-not-showing-up-in-organization-packages-ui/130481/4) for more information